src/blueteam/LumberJack.java

```diff
@@ -32,54 +33,127 @@
 		moveDirection = randomDirection();
 	}
 
+	boolean friendlyStrikeDamage() {
+		return rc.senseNearbyRobots(GameConstants.LUMBERJACK_STRIKE_RADIUS, rc.getTeam()).length == 0 ? false : true;
```

Comment:
It is correct, indeed. I just offered a code simplification. (Unnecessary ternary operator for a simple expression negation.) Sorry for nitpicking. ;-)